src/current/v23.1/monitor-and-debug-changefeeds.md

```diff
@@ -157,6 +157,39 @@ changefeed_emitted_bytes{scope="vehicles"} 183557
 `backfill_pending_ranges` | Number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) in an ongoing backfill that are yet to be fully emitted. | Ranges
 `message_size_hist` | Distribution in the size of emitted messages. | Bytes
 
+### Monitoing and measuring changefeed latency
```

Comment:
New content starts here.

Added two subsections to recommend 1. reviewing admit/commit latency percentiles 2. checking out lagging ranges